ext/RMagick/rmmain.c

```diff
@@ -1870,7 +1870,8 @@ test_Magick_version(void)
         }
 
         rb_raise(rb_eRuntimeError,
-                 "This installation of RMagick was configured with %s %s but %.*s is in use.\n" ,
+                 "This installation of RMagick was configured with %s %s but %.*s is in use.\n"
+                 "Please re-install RMagick to get rid of this exception.\n",
```

Comment:
Maybe "Please re-install RMagick to correct the issue."

Reply:
Thanks. Fixed.